server/views/pages/recommendations/selectIndexOffence.njk

```diff
@@ -32,166 +32,188 @@
                 }) }}
             {% endif %}
 
```

Comment:
Why is there so much churn on this file?  Is this all indentation?

Reply:
Its grid row - you've removed it.  line 35

Reply:
all sorted!